apps/snitch_core/lib/core/tools/helpers/query.ex

```diff
@@ -6,11 +6,16 @@ defmodule Snitch.Tools.Helper.Query do
   everywhere.
   """
 
-  @spec get(module, map | non_neg_integer, Ecto.Repo.t()) :: Ecto.Schema.t() | nil | no_return
+  @spec get(module, map | non_neg_integer | binary, Ecto.Repo.t()) ::
```

Comment:
we have `binary` id's too now ?

Reply:
We had to always parse the id's to int because Query Helper expected integer.
Just added support for string.